src/views/catalog/customize/cd.ts

```diff
@@ -25,7 +26,7 @@ export const addInstallationCDRom = (
     cdVolume = {
       name: INSTALLATION_CDROM_NAME,
       containerDisk: {
-        image: cdSource.source?.registry.url,
+        image: removeDockerPrefix(cdSource.source?.registry.url),
```

Comment:
do we add this prefix somewhere? if yes, it's better in my opinion avoid adding something not needed instead of adding than removing. if not please ignore this comment :)

Reply:
yes, we do add, i think there is a function addDockerPrefix, maybe it can be done before calling this function in case of a container disk